Source/WebCore/platform/graphics/cpu/arm/filters/FELightingNEON.h

```diff
@@ -93,14 +94,14 @@ extern "C" {
 void neonDrawLighting(FELightingPaintingDataForNeon*);
 }
 
-inline void FELighting::platformApplyNeon(const LightingData& data, const LightSource::PaintingData& paintingData)
+inline void FELightingSoftwareApplier::applyPlatformNeon(const FELightingSoftwareApplier::LightingData& data, const LightSource::PaintingData& paintingData)
```

Comment:
I think LightingData does not need the prefix FELightingSoftwareApplier since it is in its scope.